core/src/avm2/globals/flash/system/system.rs

```diff
@@ -0,0 +1,25 @@
+//! `flash.system.System` native methods
+
+use crate::avm2::activation::Activation;
+use crate::avm2::object::Object;
+use crate::avm2::value::Value;
+use crate::avm2::Error;
+
+/// Implements `flash.system.System.setClipboard` method
+pub fn set_clipboard<'gc>(
+    activation: &mut Activation<'_, 'gc, '_>,
+    _this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    //TODO: in FP9+ this function only works when called from a button handler in the Plugin due to
```

Comment:
Can you copy this comment to `fn set_clipboard_content` in web? As this will especially matter when clipboard is implemented there